cogs/displayinfo.py

```diff
@@ -33,85 +33,113 @@ class DisplayInfo(commands.Cog):
 
     def __init__(self,bot):
         self.bot = bot
-        self.timers = {}
-        self.current_collection = ""
+        self.current_collection = db["UserTracking"] # TODO: Move this secrets file
+        self.timers = self.current_collection.find_one() 
         self.current_guild_name = ""
         self.current_date = str(date.today())
+        self.members_online = set()
+        self.timed_task = None
+    
+    @commands.Cog.listener()
+    async def on_ready(self):
+        for guild in self.bot.guilds:
+            for member in guild.members:
+                if member.status == discord.Status.online:
+                    self.members_online.add(member)
```

Comment:
besides adding members to the set if they are doing an activity, we should also update them now so if the bot just came online, it can start tracking members that are already in an activity.